sentry_streams/sentry_streams/runner.py

```diff
@@ -55,22 +55,14 @@ def iterate_edges(
                     step_streams[branch_name] = next_step_stream[branch_name]
 
 
-def runner(
-    name: str,
-    log_level: str,
+def load_runtime(
     adapter: str,
     config: str,
     segment_id: Optional[str],
     application: str,
-) -> None:
+) -> Any:
```

Comment:
Please feel free to add the right type that is an adapter. 
Most likely you will run into generic issues, then you can cast it to the type you need. It will still be better than Any